website/www/site/content/en/blog/beam-2.11.0.md

```diff
@@ -58,9 +58,9 @@ For more information on changes in 2.11.0, check out the
 * Java: grpc_protobuf_lite: 1.17.1
 * Java: grpc_netty: 1.17.1
 * Java: grpc_stub: 1.17.1
-* Java: netty_handler: 4.1.30.Final
-* Java: netty_tcnative_boringssl_static: 2.0.17.Final
-* Java: netty_transport_native_epoll: 4.1.30.Final
+* Java: netty_handler: 4.1.51.Final
+* Java: netty_tcnative_boringssl_static: 2.0.33.Final
+* Java: netty_transport_native_epoll: 4.1.51.Final
```

Comment:
This is a blog post about Beam 2.11 so I think we want to keep the old version numbers around.
```suggestion
* Java: netty_handler: 4.1.30.Final
* Java: netty_tcnative_boringssl_static: 2.0.17.Final
* Java: netty_transport_native_epoll: 4.1.30.Final
```